### ISSUE TITLE
KDE Konsole Icons Not Showing Hack

### ISSUE THREAD

**Issue by JamesFraserLogan** [ryanoasis/nerd-fonts#1331]
#### 🗹 Requirements

- [ YES] I have searched the [issues](https://github.com/ryanoasis/nerd-fonts/issues) for my issue and found nothing related and/or helpful
- [ YES] I have searched the [FAQ](https://github.com/ryanoasis/nerd-fonts/wiki/FAQ-and-Troubleshooting) for help
- [ YES] I have searched the [Wiki](https://github.com/ryanoasis/nerd-fonts/wiki) for help

#### 🎯 Subject of the issue

Experienced behavior:

## Apology

Apologies, I did not read carefully enough the first time. 

### Aside

I know how to customize Konsole, change the fonts etc I just wanted to give nerd fonts a try so I could get more icons (you rule!). I can't provide screen shots until Friday; but all the non-icon chars work 100%; no difference! Perfect! Just no icons such as snake 🐍. 

## Executive Summary

In kde konsole, in ubuntu 22.04 server, the Hack nerd-font will install as per usual; That is, one can right-click install and then tell konsole to use it. The font works 100% for everything that the old font did. However, icons are not showing.

I reloaded the terminal a few times and rebooted once; but I did not explicitly stop the session ( I was reading up more in the past couple days).

## Detailed Summary

I downloaded Hack Nerd Font from https://github.com/ryanoasis/nerd-fonts/releases/download/v3.0.2/Hack.zip. I then unzipped it and right-click installed; it worked and I didn't even have to refresh the font cache (it does it for you). I installed it system-wide; it works! I can see it's where it should be, and even better; Konsole sees it and allows me to select it. And it's showing up properly under "monospace" fonts (which I need).

 HackNerdFontMono-Regular.ttf is the font I am selecting. I use a size 10.

Expected behavior:

I expected to see icons such as the snake.
Example symbols:

🐍  the snake will not show up. No icon will show up.
#### 🔧 Your Setup

- _Which font are you using (e.g. `Anonymice Powerline Nerd Font Complete.ttf`)?_
   - _Please give the full filename_

I am using  HackNerdFontMono-Regular.ttf (i made a markdown copy box I hope)

```HackNerdFontMono-Regular.ttf```

- _Where did you get the file from (download link, self patched, source downloaded from link...)_
   -  https://github.com/ryanoasis/nerd-fonts/releases/download/v3.0.2/Hack.zip

- _Which terminal emulator are you using (e.g. `iterm2`, `urxvt`, `gnome`, `konsole`)?_
   -  Konsole.
- _Are you using OS X, Linux or Windows? And which specific version or distribution?_
   -  Ubuntu server 22.04, kde neon with the full desktop package, sddm.

#### ★ Screenshots (Optional)

I am sorry I can't get access to them right now. Thanks very much for your time and effort; I very much appreciate it.

**Comment by Finii** [comment]
Ok.

You specifically mention the snake, codepoint `1F40D`.

That icon is not part of the Nerd Font icons. Usually you get that via NotoColorEmoji or NotoEmoji-Regular.

The only snake present in Nerd Fonts is this:

![image](https://github.com/ryanoasis/nerd-fonts/assets/16012374/542160b5-a72c-4aef-a65d-3a9660ac4791)
_From https://www.nerdfonts.com/cheat-sheet_

Please check if the icons you miss are really part of Nerd Fonts' patch-set. For this you can type the codepoint in hex into the cheat-sheet or copy and paste the symbol (on it's own, no extra other characters) into it:

![image](https://github.com/ryanoasis/nerd-fonts/assets/16012374/9048ca08-9269-4eda-a9b1-d7488dfd0241)

**Comment by JamesFraserLogan** [comment]
Hi Finii,

Thanks so much for your response, it is much appreciated. I will be able to test this tomorrow and will get back to you as soon as I do.

I very much appreciate your response and time in this awesome project. And you are right, I believe I was able to get the snake I posted with noto-emoji, and I assumed it was also covered in nerd fonts.

I thought I did a bunch of icons in addition to the snake; I just have to double check when I get access to the machine in question tomorrow.

There's a good chance this is just me not being thorough enough and conflating the snake icons and I will make sure to post either way.

I really love that people like you and the nerdfont family exist and are improving the lives of other shell users and developers; thank you very much. I don't know if my skill set would allow me to contribute in a meaningful way to this great project, but I will be looking into it to try and repay you wonderful people for your hard and diligent work.

Best Regards,

-James Logan

**Comment by LeoniePhiline** [comment]
I also use Konsole and found an icon missing:

![grafik](https://github.com/ryanoasis/nerd-fonts/assets/22329650/ec9d4782-a7ef-4a81-aacf-8be0fb3e190e)

Copy and pasting the replacement character (boxed question mark) into the cheat sheet, I learned that the icon has been removed, so its not showing properly is expected:

![grafik](https://github.com/ryanoasis/nerd-fonts/assets/22329650/2f29b54d-038e-4755-a532-30ffc176b932)

Maybe you happened to have the same case? Could you check if your affected icons got removed?

**Comment by Finii** [comment]
@LeoniePhiline See #1190 and possibly use https://github.com/loichyan/nerdfix

That is a breaking change introduced with Nerd Fonts v3.

**Comment by JamesFraserLogan** [comment]
Hi Finii,

Sorry for the delay, I was unable to re-create the bug. I then installed a ubuntu server 22.04 lts vm with kde-plasma-desktop installed and installed the nerdfont "hacker" font again; monospace regular. 

I got a new temporary bug where the font wouldn't show up after install, when it showed up last time but I couldn't get icons on Konsole. I restarted the machine a couple of times and it worked 100%.

I think my session manager may have malfunctioned and didn't restart on reboot. I did a pretty harsh "sudo reboot" and that's probably why it didn't restart nicely.

Thanks so much for your time and attention and I apologize for wasting your time.

Best Regards,

-James Logan
